packages/api-specification/interface.ts

```diff
@@ -583,6 +583,23 @@ declare namespace ssf {
      */
     body?: string;
   }
+
+  interface NotificationPermissionCallback {
+    (permission: String): void;
+  }
+
+  class Notification {
+    /**
+     * Create a notification
+     * @param title - The title text of the notification.
+     * @param options - The notification options.
+     */
+    constructor(title: string, options: NotificationOptions);
+    
+    static permission: string;
+    static requestPermission(callback: (NotificationPermissionCallback));
```

Comment:
Looking on MDN the callback signature is deprecated. How about using the promise-based signature instead?

https://developer.mozilla.org/en-US/docs/Web/API/Notification/requestPermission